creator-node/test/lib/blacklistManager.js

```diff
@@ -0,0 +1,30 @@
+const BlacklistManager = require('../../src/blacklistManager')
+
+const { deleteKeyPatternInRedis } = require('./utils')
+
+/**
+ * Removes all BlacklistManager keys in redis
+ * @param {Object} redis
+ */
+async function restartBlacklistManager(redis) {
```

Comment:
a couple questions --
1. does something ever get unblacklisted and require deleting all these keys outside of these tests?
2. it looks like this is only used in `afterEach` so why not just do `await appInfo.app.get('redisClient').flushdb()` in `beforeEach`? it would clear the database just like this does, but it's a one-liner instead of a whole util that requires pipelining and scanning (which look solid otherwise)

Reply:
great Qs

1. generally no, but that may happen. we have explicit code that allows content to be removed if necessary. this fn was purely for clearing redis for BL keys, which we dont need anymore on init
2. that's true! we could just do that. just for the sake of scope, I only wanted to remove what was necessary. in this case, it would only be the bl stuff

Reply:
I don't think we should ever need other data in redis between tests anyway (and I've done flushdb() in other tests with no issues) so I'd prefer not having this extra stuff. but it's a pretty cool util function and you already did the work of making it so 🤷‍♂️ I guess it'll be a nice shared util following Dheeraj's comment

Reply:
yeah thats fair, state should not carry over into other tests. so theoretically, youre right! an entire redis dump should solve this problem. I think we can keep this fn since it might be useful in the future and this test as is